src/Text/LLVM/PP.hs

```diff
@@ -980,9 +980,24 @@ ppConstExpr' pp expr =
                        in reverse  -- ppTyp's pushes entries to the listg head
                           $ foldl (ppTyp's mrng) [ppType ty]
                           $ zip argIndices (ptr:ixs)))
-    ConstConv op tv t  -> ppConvOp op <+> parens (ppTyp' tv <+> "to" <+> ppType t)
+    ConstConv op tv t  ->
+      let droppedIn18 = case op of
+                          ZExt _ -> True
+                          SExt -> True
```

Comment:
These two were removed in https://github.com/llvm/llvm-project/commit/e4a4122eb6768b69640173b4c32fd88de4547227.